### PR TITLE
Adding missing test to master branch

### PR DESCRIPTION
**Pull request by utka**

### CODE REVIEW

File: prow/cluster/jobs/istio/istio/istio.istio.master.yaml [hunk 1]
```diff
@@ -270,6 +270,22 @@ presubmits:
           - prow/e2e-simpleTests.sh
         nodeSelector:
           testing: test-pool
+    - name: e2e-simpleTestsMinProfile-master
+      <<: *job_template
+      optional: true
```
Comment (icygalz) [inline prow/cluster/jobs/istio/istio/istio.istio.master.yaml:275]: just a question, can you tell me the difference between "always_run" and "optional" keyword?

File: prow/cluster/jobs/istio/istio/istio.istio.master.yaml [hunk 2]
```diff
@@ -374,6 +390,20 @@ postsubmits:
           - prow/istio-integ-k8s-tests.sh
         nodeSelector:
           testing: test-pool
+    - name: e2e-simpleTestsMinProfile-master
+      <<: *job_template
+      optional: true
```
Comment (icygalz) [inline prow/cluster/jobs/istio/istio/istio.istio.master.yaml]: is this flag needed for postsubmit test? Other postsubmit tests don't seem to have that field.

File: prow/cluster/jobs/istio/istio/istio.istio.master.yaml (hunk 1 above)
Comment (utka) [reply prow/cluster/jobs/istio/istio/istio.istio.master.yaml:275]: Optional means required test or not.

File: prow/cluster/jobs/istio/istio/istio.istio.master.yaml (hunk 2 above)
Comment (utka) [reply prow/cluster/jobs/istio/istio/istio.istio.master.yaml]: Interesting, I copied it from release 1.1 file, will remove from both

Comment (utka) [reply prow/cluster/jobs/istio/istio/istio.istio.master.yaml]: Actually I do not want to change anything for release 1.1. Just removed it for master

**Pull request merged**